src/uwtools/tests/test_cli.py

```diff
@@ -10,6 +10,11 @@
 import pytest
 from pytest import fixture, raises
 
+import uwtools.api.config
+import uwtools.api.forecast
+import uwtools.api.rocoto
+import uwtools.api.template
```

Comment:
The test of the CLI dispatch functions now essentially just check that they are wired up to the API functions correctly.